generators/entity-server/templates/src/main/kotlin/package/web/rest/EntityResource.kt.ejs

```diff
@@ -218,6 +222,7 @@ _%><%- include('../../common/inject_template', {viaService: viaService, construc
      * @throws URISyntaxException if the Location URI syntax is incorrect.
      */
     @PutMapping("/<%= entityApiUrl %>/{<%= primaryKey.name %>}")
+    @Suppress("ThrowsCount")
```

Comment:
can't we globally disable these ? rather than changing the values in the template I would like to change the configuration to allow these, wdyt?

Reply:
I don't have a strong opinion. This rule in particular is unnecessarily strict IMOP, so no personal taste issue with disabling. The user can reenable and fix if they wish.